### ISSUE TITLE
Non-whitelisted users get stuck when proposing milestones

### ISSUE THREAD

**Issue by vsimetka** [Giveth/giveth-dapp#185]
Non-whitelisted users can access the propose milestone page, they can fill in the form and submit the milestone getting a positive confirmation. However, they get stuck on this page and the milestone actually is not proposed.

**Solution:**
Non-whitelisted users should not have access to the propose milestone form and should not even see the propose milestone button.

![screen shot 2017-12-01 at 22 47 34](https://user-images.githubusercontent.com/7974813/33511554-984f4cf8-d6ea-11e7-9b65-2d66a4ad0421.png)

<img width="866" alt="screen shot 2017-12-01 at 22 48 29" src="https://user-images.githubusercontent.com/7974813/33511556-9d7479f6-d6ea-11e7-8677-b3de8a59a283.png">

**Comment by perissology** [comment]
I don't think we need a whitelist for milestones. It seems to make sense to allow anyone to propose milestones, especially since a milestone has to be approved by the campaign owner before it is created.

see #156 

feathers needs to be updated to fix this issue
